src/App.js

```diff
@@ -470,23 +488,77 @@ class _AppStackWrapper extends React.Component {
 
 const AppStackWrapper = connect(mapStateToProps, mapDispatchToProps)(_AppStackWrapper);
 
-const SwitchNavigator = createSwitchNavigator({
-  Decide: DecideStackScreen,
-  App: AppStackWrapper,
-  Init: InitStack,
-}, {
-  initialRouteName: 'Decide',
-});
+const BlankScreen = () => null;
+
+const RootStack = () => {
```

Comment:
🚲 I think we should have a document string here like we see in InitStack.

Reply:
Done on [ae88d09](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/286/commits/ae88d09c6cdb386a0025367f7f7a6713205f4e72)